scripts/pi-hole/js/utils.js

```diff
@@ -230,6 +230,15 @@ function getGraphType() {
   return localStorage.getItem("barchart_chkbox") === "false" ? "line" : "bar";
 }
 
+function textToClipboard(text) {
+  var tmpTextarea = document.createElement("textarea");
+  document.body.appendChild(tmpTextarea);
+  tmpTextarea.value = text;
+  tmpTextarea.select();
+  document.execCommand("copy");
```

Comment:
Thanks for your suggestion. However, I've just found

> This feature [`document.execCommand`] is obsolete. Although it may still work in some browsers, its use is discouraged since it could be removed at any time. Try to avoid using it.

https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand

and

> The `execCommand('copy')` API isn't supported in Safari

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard

There seems to be the new asynchronous Clipboard API methods to add this feature, however, ...

> The asynchronous Clipboard API methods are a recent addition to the specification, and may not be fully implemented to the specification in all browsers. Be sure to review the compatibility tables for each method before using them, to ensure that support is broad enough for your needs.

(same source as above)

Reply:
Compatibility matrix: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API

![Screenshot_20201011-202027](https://user-images.githubusercontent.com/16748619/95686628-583eaa80-0bff-11eb-9892-c80bfa7ba27c.png)

Reply:
Oh. Fiddlesticks...

Reply:
I'll look at using this newer API. Maybe it's time we say "to hell with internet explorer", anyway

Reply:
That said, the clipboard API is still in working draft status. So probably best not to use it....